### PR TITLE
Test cross compiling with cargo-zigbuild

### PR DESCRIPTION
**Pull request by messense**
https://github.com/messense/cargo-zigbuild

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -170,7 +170,7 @@ jobs:
         run: |
           cd examples/rust_with_cffi/
           python --version
-          pip install -U pip wheel setuptools
```
Comment (messense) [inline .github/workflows/ci.yml:173]:
```
ERROR: Could not install packages due to an OSError: [WinError 5] Access is denied: 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-uninstall-d0uy_7qf\\pip.exe'
Consider using the `--user` option or check the permissions.
```

Upgrade pip on Windows fails with permission error.

Comment (davidhewitt) [reply .github/workflows/ci.yml:173]: I think you can get around this by using `python -m pip` instead of `pip` as the executable.

Comment (messense) [reply .github/workflows/ci.yml:173]: Well, I think it's not needed to be upgraded now.

File: README.md [hunk 2]
```diff
@@ -79,7 +79,8 @@ Finished processing dependencies for hello_rust==1.0
 
 Or you can use commands like `bdist_wheel` (after installing `wheel`). See also [the notes in the documentation about building wheels](https://setuptools-rust.readthedocs.io/en/latest/building_wheels.html).
 
-Cross-compiling is also supported, using either [`crossenv`](https://github.com/benfogle/crossenv) or [`cross`](https://github.com/rust-embedded/cross). For examples see the `test-cross-compile` and `test-cross` Github actions jobs in [`ci.yml`](https://github.com/PyO3/setuptools-rust/blob/main/.github/workflows/ci.yml).
+Cross-compiling is also supported, using either [`crossenv`](https://github.com/benfogle/crossenv) or [`cross`](https://github.com/rust-embedded/cross) or [`cargo-zigbuild`](https://github.com/messense/cargo-zigbuild).
```
Comment (adamreichold) [inline README.md:82]:
```suggestion
Cross-compiling is also supported, using one of [`crossenv`](https://github.com/benfogle/crossenv), [`cross`](https://github.com/rust-embedded/cross) or [`cargo-zigbuild`](https://github.com/messense/cargo-zigbuild).
```

**Pull request merged**